app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -214,6 +216,14 @@ public void onClick(View v) {
                 startActivityForResult(i, LOCATION_EDIT);
             }
         });
+        spinner = findViewById(R.id.Marker_selector);
+        spinner.setOnItemSelectedListener(new SimpleOnItemSelectedListener() {
+            @Override
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+                super.onItemSelected(parent, view, position, id);
```

Comment:
`super.onItemSelected(...);` can be removed because it is not necessariy. `SimpleOnItemSelectedListener` has no code in the implementation so it does literally nothing.

Reply:
done